project/core/urls.py

```diff
@@ -30,4 +32,12 @@
     path("caregivers/", include("caregivers.urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("people/", include("people.urls")),
+    
+    path('password_change/done/', auth_views.PasswordChangeDoneView.as_view(template_name='registration/password_change_complete.html'), 
+        name='password_change_done'),
```

Comment:
View names are, by convention, hyphenated in this project. Let's switch to using hyphens instead of underscores.
```suggestion
        name='password-change-done'),
```